app/src/main/kotlin/no/nav/tiltakspenger/vedtak/clients/oppgave/OppgaveDto.kt

```diff
@@ -1,23 +1,51 @@
 package no.nav.tiltakspenger.vedtak.clients.oppgave
 
+import no.nav.tiltakspenger.felles.journalføring.JournalpostId
+import no.nav.tiltakspenger.libs.common.Fnr
 import java.time.DayOfWeek
 import java.time.LocalDate
 
 const val TEMA_TILTAKSPENGER: String = "IND"
 const val OPPGAVETYPE_BEHANDLE_SAK: String = "BEH_SAK"
+const val OPPGAVETYPE_VURDER_KONSEKVENS_FOR_YTELSE: String = "VUR_KONS_YTE"
+
+const val BEHANDLES_AV_APPLIKASJON = "TILTAKSPENGER"
 
 data class OpprettOppgaveRequest(
     val personident: String,
     val opprettetAvEnhetsnr: String = "9999",
-    val journalpostId: String,
-    val behandlesAvApplikasjon: String = "TILTAKSPENGER",
-    val beskrivelse: String = "Ny søknad om tiltakspenger. Behandles i ny løsning.",
+    val journalpostId: String?,
+    val behandlesAvApplikasjon: String?,
+    val beskrivelse: String,
     val tema: String = TEMA_TILTAKSPENGER,
-    val oppgavetype: String = OPPGAVETYPE_BEHANDLE_SAK,
+    val oppgavetype: String,
     val aktivDato: LocalDate = LocalDate.now(),
     val fristFerdigstillelse: LocalDate = finnFristForFerdigstillingAvOppgave(LocalDate.now().plusDays(3)),
     val prioritet: PrioritetType = PrioritetType.NORM,
-)
+) {
+    companion object {
+        fun opprettOppgaveRequestForSoknad(
+            fnr: Fnr,
+            journalpostId: JournalpostId,
+        ) = OpprettOppgaveRequest(
+            personident = fnr.verdi,
+            journalpostId = journalpostId.toString(),
+            beskrivelse = "Ny søknad om tiltakspenger. Behandles i ny løsning.",
+            behandlesAvApplikasjon = BEHANDLES_AV_APPLIKASJON,
+            oppgavetype = OPPGAVETYPE_BEHANDLE_SAK,
+        )
+
+        fun opprettOppgaveRequestForEndretTiltaksdeltaker(
+            fnr: Fnr,
+        ) = OpprettOppgaveRequest(
+            personident = fnr.verdi,
+            journalpostId = null,
+            beskrivelse = "Det har skjedd en endring i tiltaksdeltakelsen som kan påvirke tiltakspengeytelsen.",
```

Comment:
Oppgaveteksten for disse oppgavene vil jeg at Taulant skal se på når vi tester i Gosys, og så kan jeg evt endre etterpå. Jeg har også latt oppgaven være en vanlig Gosys-oppgave siden det ikke er sikkert den skal behandles hos oss, men det vil jeg også gjerne se på sammen med Taulant når det ligger i dev (det er uansett lett å endre på etterpå)